config-server/src/main/resources/application.properties

```diff
@@ -1,5 +0,0 @@
-server.port=8888
```

Comment:
this file should not be deleted. it should be refactored into application.yml

Reply:
Fixed